build.sh

```diff
@@ -92,7 +92,7 @@ do
       (cd lang/csharp; ./build.sh test)
       (cd lang/js; ./build.sh lint test)
       (cd lang/ruby; ./build.sh lint test)
-      (cd lang/php; ./build.sh test)
+      (cd lang/php; ./build.sh lint test)
```

Comment:
Ooops, this breaks on a **clean** build on my system!

```
+ cd lang/php
+ ./build.sh lint test
No syntax errors detected in ./test/IODatumReaderTest.php
No syntax errors detected in ./test/test_helper.php
No syntax errors detected in ./examples/write_read.php
No syntax errors detected in ./test/ProtocolFileTest.php
No syntax errors detected in ./test/InterOpTest.php
No syntax errors detected in ./test/FloatIntEncodingTest.php
No syntax errors detected in ./test/LongEncodingTest.php
No syntax errors detected in ./test/StringIOTest.php
No syntax errors detected in ./test/generate_interop_data.php
No syntax errors detected in ./test/NameTest.php
No syntax errors detected in ./test/DataFileTest.php
No syntax errors detected in ./lib/AvroIO.php
No syntax errors detected in ./test/SchemaTest.php
No syntax errors detected in ./test/DatumIOTest.php
No syntax errors detected in ./lib/Protocol/AvroProtocolParseException.php
No syntax errors detected in ./lib/Protocol/AvroProtocol.php
No syntax errors detected in ./lib/AvroNotImplementedException.php
No syntax errors detected in ./lib/Protocol/AvroProtocolMessage.php
No syntax errors detected in ./lib/AvroUtil.php
No syntax errors detected in ./lib/autoload.php
No syntax errors detected in ./lib/Schema/AvroSchemaParseException.php
No syntax errors detected in ./lib/Schema/AvroMapSchema.php
No syntax errors detected in ./lib/Schema/AvroArraySchema.php
No syntax errors detected in ./lib/Schema/AvroName.php
No syntax errors detected in ./lib/Schema/AvroFixedSchema.php
No syntax errors detected in ./lib/Schema/AvroSchema.php
No syntax errors detected in ./lib/Schema/AvroRecordSchema.php
No syntax errors detected in ./lib/Schema/AvroUnionSchema.php
No syntax errors detected in ./lib/Schema/AvroNamedSchemata.php
No syntax errors detected in ./lib/Schema/AvroField.php
No syntax errors detected in ./lib/Schema/AvroEnumSchema.php
No syntax errors detected in ./lib/Schema/AvroPrimitiveSchema.php
No syntax errors detected in ./lib/Schema/AvroNamedSchema.php
No syntax errors detected in ./lib/DataFile/AvroDataIOException.php
No syntax errors detected in ./lib/DataFile/AvroDataIOWriter.php
No syntax errors detected in ./lib/DataFile/AvroDataIO.php
No syntax errors detected in ./lib/DataFile/AvroDataIOReader.php
No syntax errors detected in ./lib/IO/AvroStringIO.php
No syntax errors detected in ./lib/IO/AvroFile.php
No syntax errors detected in ./lib/IO/AvroIOException.php
No syntax errors detected in ./lib/Avro.php
No syntax errors detected in ./lib/Datum/AvroIOSchemaMatchException.php
No syntax errors detected in ./lib/AvroGMP.php
No syntax errors detected in ./lib/Datum/AvroIOBinaryEncoder.php
No syntax errors detected in ./lib/Datum/AvroIODatumWriter.php
No syntax errors detected in ./lib/Datum/AvroIOBinaryDecoder.php
No syntax errors detected in ./lib/AvroDebug.php
No syntax errors detected in ./lib/Datum/AvroIOTypeException.php
No syntax errors detected in ./lib/AvroException.php
No syntax errors detected in ./lib/Datum/AvroIODatumReader.php
./build.sh: line 61: vendor/bin/phpcs: No such file or directory
```

phpcs is installed by composer, we can either run `test` before `lint` or call `composer install ...` in the [lint target](https://github.com/sekikn/avro/blob/AVRO-2975/lang/php/build.sh#L59-L67) as well (it looks like calling it twice is fast).

Reply:
Thank you for the comment @RyanSkraba! Will update the PR soon :)